MyHashSet.java

```diff
@@ -0,0 +1,58 @@
+// Time Complexity : O(1) amortized for each operation
+// Space Complexity : )(n) where n is max size of hashset
```

Comment:
Since the size here is a constant i.e 1000. The Space will be O(1000^2), which is O(1).